honeybee_grasshopper_radiance/src/HB Sky Matrix.py

```diff
@@ -0,0 +1,64 @@
+# Honeybee: A Plugin for Environmental Analysis (GPL) started by Mostapha Sadeghipour Roudsari
```

Comment:
This is the old header. The new one doesn't have my name.

Reply:
Good catch.